src/main/java/frc/robot/controls/Driver.java

```diff
@@ -2,20 +2,38 @@
 
 import edu.wpi.first.wpilibj.Joystick;
 import frc.robot.Constants;
+import java.lang.Math;
 
 public class Driver
 {
     public static Joystick throttle = new Joystick(Constants.THROTTLE_JOYSTICK);
     public static Joystick steer = new Joystick(Constants.STEER_JOYSTICK);
 
+    private static double throttleVal = 0;
+    private static double steerVal = 0;
+
```

Comment:
Why do you need to make these variables up here if they are only used inside `getSteer()` and `getThrottle()`?